src/test/java/edu/cornell/kfs/module/ezra/service/ezraUpdateAwardImplTest.java

```diff
@@ -41,10 +41,17 @@ protected void setUp() throws Exception {
     public void testUpdateAwards () {
         Map fields = new HashMap();
         String awardProjectTitle = "ezraUpdateTest";
-        String awardProjectTitleAfter = ""; 
-        java.sql.Date today = SpringContext.getBean(DateTimeService.class).getCurrentSqlDate();
+        String awardProjectTitleAfter = "";
+        Date today = SpringContext.getBean(DateTimeService.class).getCurrentSqlDate();
```

Comment:
We could create a getDateTimeService() function, but that might be beyond the scope of this user story. This unit test case already makes heavy use of SpringContext.getBean() (see the setUp() function above), and also relies on the EzraService performing actual data access to EZRA. Some significant refactoring would be necessary to turn this unit testing class into a micro testing one.

In the case of the DateTimeService above, I was simply editing an existing SpringContext.getBean() call to shorten the fully-qualified Date classname, to avoid confusion among the adjacent variables that are not fully qualified but still refer to the same class.